cgi-bin/CATS/UI/ProblemHistory.pm

```diff
@@ -111,13 +116,41 @@ sub problem_history_tree_frame {
         tree => $tree,
         problem_title => $pr->{title},
         title_suffix => $pr->{title},
+        problem_import_log => $log,
+        messages => @{CATS::Messages::get()},
```

Comment:
`messages` should be passed to template already. Are you sure this is not the case?